.travis.yml

```diff
@@ -18,4 +18,4 @@ install: true
 script:
   - make .gitvalidation
   - make lint
-  - make binaries
+  - make ocid ocic
```

Comment:
okay, no worries. Can you rebase? I will test locally and merge if everything works as expected.